### ISSUE TITLE
Problem and Solution

### ISSUE THREAD

**Issue by xvanlima** [PhocaCz/PhocaOpenGraphPlugin#1]
[error] 27417#27417: *100 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined property: JObject::$params in /home/ivan/Documentos/Projetos/puft/plugins/content/phocaopengraph/phocaopengraph.php on line 285" while reading response header from upstream

```php
@@ -276,10 +276,10 @@ class plgContentPhocaOpenGraph extends JPlugin
+               } else if ($this->params->get('menu-meta_description') != '') { // menu link meta description
+                        plgContentPhocaOpenGraphHelper::renderTag('og:description', $this->params->get('menu-meta_description'), $type);
-               } else if ($row->params->get('menu-meta_description') != '') { // menu link meta description
-                        plgContentPhocaOpenGraphHelper::renderTag('og:description', $row->params->get('menu-meta_description'), $type);
```

**Comment by PhocaCz** [comment]
Hi, thank you, will be fixed in next version.

**Comment by PhocaCz** [comment]
Fixed in version 3.0.9
https://www.phoca.cz/news/941-phoca-open-graph-content-3-0-9-released
